backend/tilavarauspalvelu/api/graphql/types/recurring_reservation/serializers/add_serializer.py

```diff
@@ -102,6 +103,10 @@ def update(self, instance: RecurringReservation, validated_data: ReservationSeri
         reservation.buffer_time_after = validated_data["buffer_time_after"]
         reservation.access_type = validated_data["access_type"]
 
+        if reservation.access_type != AccessType.ACCESS_CODE:
```

Comment:
condition not necessary; these could be the default values in any case (then updated by the subsequent reschedule)?